internal/ad/ad.go

```diff
@@ -546,11 +546,16 @@ func (ad *AD) GetStatus() (adServerURL string, isOffline bool) {
 	return ad.url, ad.isOffline
 }
 
-// NormalizeTargetName transform user or domain\user as user@domain.
+// NormalizeTargetName transforms User or DOMAIN\User to user@domain.
```

Comment:
I would add in the description that we lowercase it. But that’s the right place to change :)